pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "contentrepo"
-version = "1.1.0-dev.3"
+version = "2.0.0-dev.0"
```

Comment:
Do we really need to bump the major version for this?

Reply:
Any existing services using the API will be looking at `next_prompt` in the API, they need to change to look in the `buttons` section instead, in a completely different format (there may be multiple buttons, they might not all be next buttons). So I would say for the API users, it is a major change.

The other thing we could do is version the API. So do something like the existing API returns the first next message button as `next_prompt`, and then we have a new version that returns the full `buttons` detail. I'm not sure if that's worth the effort though.